package.json

```diff
@@ -270,12 +270,12 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run compile",
-    "package": "vsce package",
-    "compile": "tsc -b",
-    "watch": "tsc -b -w",
-    "lint": "eslint src/**/*.ts",
-    "test-compile": "tsc -p ./",
-    "test": "npm run test-compile && node ./out/test/runTest.js"
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
+    "lint": "eslint src --ext ts",
+    "pretest": "npm run compile && npm run lint",
```

Comment:
What is the use case for the `pretest` task?

Reply:
Another thing that comes from the default `yeoman` templates. Anything prepended with `pre` gets run before the similarly named script. So `pretest` gets run before `test. This reminds me I forgot to remove `npm compile` from `test.